### PR TITLE
the bubble filter finally works!

### PR DESCRIPTION
**Pull request by kreier**

### CODE REVIEW

File: art.js [hunk 1]
```diff
@@ -203,12 +203,45 @@ function setup() {
...
+      noStroke()
+      fill(dotColor)
+      ellipse(centerX, centerY, dotDiameter, dotDiameter)
+    }
+  }
+  copy_to_buffer();
+  buffer.loadPixels();
+  drawMosaic(5, color(30, 30, 30)); // radius, backgroundcolor
+  
   // Khang
```
Comment (kreier) [inline art.js:245]: Hopefully Khang's code from below here stays.

File: art.js [hunk 2]
```diff
@@ -136,7 +136,7 @@ function setup() {
   // *         2022-11-25          *
   // *******************************
   //
-  // ################### William #######################
+  // ########################## William ################################
```
Comment (matthiaskreier) [inline art.js:139]: Fix this to 80 characters

File: art.js [hunk 3]
```diff
@@ -203,12 +203,45 @@ function setup() {
...
+      dotColor = buffer.get(centerX, centerY);
+      // dotColor = example.get(centerX, centerY);
+      noStroke()
+      fill(dotColor)
+      ellipse(centerX, centerY, dotDiameter, dotDiameter)
+    }
+  }
+  copy_to_buffer();
+  buffer.loadPixels();
+  drawMosaic(5, color(30, 30, 30)); // radius, backgroundcolor
```
Comment (matthiaskreier) [inline art.js:243]: This finally works?

File: art.js (hunk 1 above)
Comment (matthiaskreier) [inline art.js:245]: Hopefully Khang's code stays intact

**Review by matthiaskreier — APPROVED** [review]

File: art.js (hunk 1 above)
Comment (kreier) [reply art.js:245]: I'll check later

**Pull request merged**